crypto/bindings/conversion.ts

```diff
@@ -10,9 +10,12 @@ import type {
 import type * as wasmNamespace from '../../compiled/node_bindings/plonk_wasm.cjs';
 import { bigIntToBytes, bytesToBigInt } from '../bigint-helpers.js';
 
+export { createRustConversion };
```

Comment:
Nit: that should not be a change (moved up, see removed line before).

Reply:
I think it's fine, I've been adding more types and having the exports at the top is better than burying it somewhere below in the file